components/organisms/LoginForm/LoginFormContent.js

```diff
@@ -79,7 +79,9 @@ const LoginFormContent = ({ onSubmit, toggleForm, activeForm }) => {
 
   const Content = () => (
     <>
-      <StyledFormTitle>{formTitle}</StyledFormTitle>
+      <StyledFormTitle className="formToggler" id="loginFormTitle">
```

Comment:
Do we use classes or ids for expecting elements?

Reply:
For elements, which used one time i used ID. Do you think it matters on ui-tests?

Reply:
I think one approach should be used, either class or id